app/controllers/registrations_controller.rb

```diff
@@ -5,9 +5,4 @@ class RegistrationsController < Devise::RegistrationsController
   #   flash[:notice] = "You're in!"
   #   :edit_user_registration
   # end
-  #
-  # def after_inactive_sign_up_path_for(_resource)
```

Comment:
When a user signs up for an account, the default behavior is to redirect to root path, which did not exist when we first started coded this feature. That is why we added the above code to change the redirect path to a path that does exist (i.e. edit user registration). Now that we have merged changes made by the other pair in the group, we not have root path that redirects to the index page. So we no longer need these commented out lines